### DISCUSSION TITLE
articles/what-does-the-heart-sutra-say

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#138] [Comments]
https://hutusi.com/articles/what-does-the-heart-sutra-say 

 《心经》全称《般若波罗蜜多心经》，是大乘佛教的经典。心经很短，只有260字，我读过很多遍，可以流利的背出，但对其中内容并没有深刻理解。无意中发现了比尔·波特《心经解读》这本书，读完有种豁然开朗的感觉，对经文中的词都有详细描述。作者比尔·波特本人是位汉学家，对中国文化和佛教文化有很深的理解，他的笔名是“赤松居士”，有古代文人之风。

**Comment by surmon-china** [comment]
我这儿也做过一个粗浅的翻译：https://surmon.me/article/183 😅

**Comment by hutusi** [comment]
> 我这儿也做过一个粗浅的翻译：https://surmon.me/article/183 😅

这翻译很现代化😀

**Comment by ArcPi** [comment]
同感！读了一些书籍，灵修、冥想、佛教的。发现其实都是讲一回事，活在当下。

**Comment by weekend-project-space** [comment]
我们理解和感受到的世界不是真实的世界，只是接近世界的一个界面，它是一套模型，在我们大脑中运作，即五蕴，受制于五蕴是无明，修行之道再于不要被五蕴裹携

是这样吗？

**Reply by hutusi** [reply]
我们改变不了世界，只能改变我们对世界的认知和感受。
